src/components/tooltip/bl-tooltip.test.ts

```diff
@@ -0,0 +1,116 @@
+import { assert, elementUpdated, expect, fixture, html } from '@open-wc/testing';
+import { sendMouse } from '@web/test-runner-commands';
+import BlTooltip from './bl-tooltip';
+import type typeOfBlTooltip from './bl-tooltip';
+
+
+describe('bl-tooltip', () => {
+  it('should be defined tooltip instance', () => {
+    //when
+    const el = document.createElement('bl-tooltip');
+
+    //then
+    assert.instanceOf(el, BlTooltip);
+  });
+
+  it('should be rendered with default values', async () => {
+    //when
+    const el = await fixture<typeOfBlTooltip>(html`<bl-tooltip></bl-tooltip>`);
+
+    //then
+    assert.shadowDom.equal(
+      el,
+      `
+      <slot 
+       class="trigger" 
+       name="tooltip-trigger">
+      </slot>
+      <div class='hidden tooltip'>  
+        <slot></slot>
+        <div class="arrow"></div>
+      </div>
+      `
+    );
+  });
+
+  it('should have correct default values', async () => {
+    //when
+    const el = await fixture<typeOfBlTooltip>(html`<bl-tooltip>Test</bl-tooltip>`);
+
+    //then
+    expect(el.placement).to.equal('top');
+  });
+
+  it('should be rendered with slot', async () => {
+    //when
+    const el = await fixture<typeOfBlTooltip>(html`<bl-tooltip><bl-button slot='tooltip-trigger'>Test</bl-button>
```

Comment:
Button component is not imported in the test. Consider importing it or not using it here.